### DISCUSSION TITLE
orm fixtures

### DISCUSSION THREAD

**Discussion by grosfaignan** [symfony/symfony#44411] [Q&A]
hi,

i tried to install` orm-fixtures`, who dont works with php 8.
do you know fixtures package who work with ?

regards

**Comment by grosfaignan** [comment]
php 8.1 sorry

**Comment by chapterjason** [comment]
Hey @grosfaignan I don't know if there is another one around.

But you can submit an issue here https://github.com/doctrine/DoctrineFixturesBundle/issues for the missing PHP 8.1 support.

**Comment by grosfaignan** [comment]
i think i'll go for 8.0 for the moment ...^^ 
cause i think it will be soon
